blob/blob-gcp/src/test/resources/delete-1wyfk7bmao.json

```diff
@@ -0,0 +1,25 @@
+{
+  "id" : "f6149961-1023-4bb1-902a-c3f650a5c5a2",
+  "name" : "storage_v1_b_substrate-sdk-gcp-poc1-test-bucket-versioned_o_conformance-tests_upload_happypath_versioned_path",
+  "request" : {
+    "url" : "/storage/v1/b/substrate-sdk-gcp-poc1-test-bucket-versioned/o/conformance-tests%2Fupload%2FhappyPath_versioned_Path",
+    "method" : "DELETE"
+  },
+  "response" : {
+    "status" : 204,
+    "headers" : {
+      "Alt-Svc" : "h3=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000",
+      "Server" : "UploadServer",
+      "Cache-Control" : "no-cache, no-store, max-age=0, must-revalidate",
+      "X-GUploader-UploadID" : "AAwnv3J6BCE-ay8JyAKYXE7-xn8MKpGbTAIqiP3VEdYILA6EF9OXXg8ckmNEBe-tDN_lnsLJ",
+      "Vary" : [ "Origin", "X-Origin" ],
+      "Pragma" : "no-cache",
+      "Expires" : "Mon, 01 Jan 1990 00:00:00 GMT",
+      "Date" : "Fri, 03 Oct 2025 00:48:00 GMT",
+      "Content-Type" : "application/json"
+    }
+  },
+  "uuid" : "f6149961-1023-4bb1-902a-c3f650a5c5a2",
+  "persistent" : true,
+  "insertionIndex" : 1
+}
```

Comment:
these should be in mappings, something is not right with the local configs while recording